pkg/cluster/config/v1alpha2/conversion.go

```diff
@@ -63,14 +73,27 @@ func Convert_v1alpha2_Node_To_config_Node(in *Node, out *config.Node, s conversi
 }
 
 func Convert_config_Cluster_To_v1alpha2_Config(in *config.Cluster, out *Config, s conversion.Scope) error {
-	// TODO(bentheelder): try to convert kubeadm config patches?
-	// Not sure if that makes sense since these aren't cluster wide, and we don't
-	// actually roundtrip anywho? :thinking:
+	foundFirstControlPlane := false // in v1alpha2 the first control plane is special
 	for _, node := range in.Nodes {
 		convertedNode := Node{}
 		if err := Convert_config_Node_To_v1alpha2_Node(&node, &convertedNode, s); err != nil {
 			return err
 		}
+		// the first control plane is treated specially
```

Comment:
specifically this new snippet means we roundtrip the global patches to v1alpha2

we should get people off of v1alpha2 though and we should not actually roundtrip to it, because per-node patches will NOT roundtrip correctly as they were ignored in v1alpha2